core/runtime/src/main/java/io/quarkus/runtime/BannerRecorder.java

```diff
@@ -7,19 +7,33 @@
 
 @Recorder
 public class BannerRecorder {
-    final RuntimeValue<BannerRuntimeConfig> bannerRuntimeConfig;
+    private static volatile String tipOfTheDay = null;
+
+    private final RuntimeValue<BannerRuntimeConfig> bannerRuntimeConfig;
 
     public BannerRecorder(RuntimeValue<BannerRuntimeConfig> bannerRuntimeConfig) {
         this.bannerRuntimeConfig = bannerRuntimeConfig;
     }
 
     public RuntimeValue<Optional<Supplier<String>>> provideBannerSupplier(String bannerText) {
         if (bannerRuntimeConfig.getValue().enabled) {
+            if (bannerRuntimeConfig.getValue().tips && bannerText.contains("${title}") && bannerText.contains("${message}")
+                    && tipOfTheDay != null) {
+                bannerText = bannerText.replace("${title}", "\uD83D\uDCA1 TIP OF THE DAY:");
```

Comment:
We will need to check how this looks like on Windows as we had a few bad surprises.

Reply:
Let me try on in a VM

Reply:
Correct, at least in the power shell, the emoji is not displayed correctly. I'll add a check not to display if it is Windows.